src/LanguageServer/Protocol/Handler/Diagnostics/DiagnosticSourceProviders/DiagnosticSourceManager.cs

```diff
@@ -106,7 +105,6 @@ public static ImmutableArray<IDiagnosticSource> AggregateSourcesIfNeeded(Immutab
         if (isDocument)
         {
             // Group all document sources into a single source.
-            Debug.Assert(sources.All(s => s.IsLiveSource()), "All document sources should be live");
```

Comment:
this was the assert that was firing.  Now there is no concept of 'live' vs. non-live sources, just live vs. non-live diagnostics